src/vstart.sh

```diff
@@ -288,6 +288,7 @@ case $1 in
 	    ;;
     --memstore )
 	    memstore=1
+	    bluestore=0
```

Comment:
@yangdongsheng instead of flipping the `memstore` and `bluestore`, i think we can even simplify this script a little bit, see #17749.